frontend/coffee/stats.coffee

```diff
@@ -1,50 +1,81 @@
+Chart = require('chart.js')
+
 window.activateStats = () ->
     worker = new Worker("/static/statworker.js")
     worker.addEventListener('message', (e) ->
         switch e.data.action
 
             when "downloads_ready"
-                new Chart(document.getElementById('downloads-over-time').getContext("2d")).Line({
-                    labels : e.data.data.labels,
-                    datasets : e.data.data.entries
-                },
-                {
-                    animation: false
-                })
-                keyUI = document.getElementById('downloads-over-time-key')
-                for k in e.data.data.key
-                    li = document.createElement('li')
-                    keyColor = document.createElement('span')
-                    keyText = document.createElement('span')
-                    keyColor.className = 'key-color'
-                    keyText.className = 'key-text'
-                    keyColor.style.backgroundColor = k.color
-                    keyText.textContent = k.name
-                    li.appendChild(keyColor)
-                    li.appendChild(keyText)
-                    keyUI.appendChild(li)
+                new Chart(document.getElementById('downloads-over-time'),
+                    type: 'line'
+                    data:
+                        labels: e.data.data.labels
+                        datasets: e.data.data.entries
+                    options:
+                        legend:
+                            fullWidth: true
+                            position: 'bottom'
+                            labels:
+                                padding: 5
+                                fontSize: 9
+                                boxWidth: 12
+                        scales:
+                            yAxes: [
+                                ticks:
+                                    precision: 0
+                            ]
+                )
 
             when "downloads_per_version_ready"
-                new Chart(document.getElementById('downloads-per-version').getContext('2d')).Bar({
-                    labels: e.data.data.labels,
-                    datasets: e.data.data.entries
-                },
-                {
-                    animation: false
-                })
+                new Chart(document.getElementById('downloads-per-version'),
+                    type: 'bar'
+                    data:
+                        labels: e.data.data.labels
+                        datasets: e.data.data.entries
+                    options:
+                        legend:
+                            display: false
+                        scales:
+                            yAxes: [
+                                ticks:
+                                    precision: 0
```

Comment:
Apparently the charts don't begin at zero by default.
Looking at the docs we'll either need `beginAtZero` or `suggestedMin`, though it sounds like the first one is a better fit.
Probably also applies to the other two charts.

https://www.chartjs.org/docs/latest/axes/cartesian/linear.html#tick-configuration-options

Reply:
Good point, and I think the perfect setting is one jump away:

> These options extend the common tick configuration.

- https://www.chartjs.org/docs/latest/axes/cartesian/#tick-configuration

min | number |   | User defined minimum value for the scale, overrides minimum value from data.
-- | -- | -- | --

The bottom chart needs to allow negatives because people can un-follow.

Reply:
The bottom chart would be a good fit for `suggestedMin` then? It would still be cool if it started at zero if the follower delta has been >0 each day the last 30 days.
And if there was a days with <0, suggestedMin is not applied if I read the docs correctly:
`let minDataValue = Math.min(mostNegativeValue, options.ticks.suggestedMin);`

Reply:
Makes sense to me.